x-pack/plugins/security_solution/common/endpoint/service/trusted_apps/validations.ts

```diff
@@ -57,10 +50,18 @@ export const hasSimpleExecutableName = ({
   type: TrustedAppEntryTypes;
   value: string;
 }): boolean => {
-  if (type === 'wildcard') {
-    return os === OperatingSystem.WINDOWS ? WIN_EXEC_PATH.test(value) : UNIX_EXEC_PATH.test(value);
+  if (type !== 'wildcard') {
+    return true;
   }
-  return true;
+
+  const separator = os === OperatingSystem.WINDOWS ? '\\' : '/';
+  const lastString = value.split(separator).pop();
+
+  if (!lastString) {
+    return false;
+  }
+
+  return /[\*\?]/.test(lastString) === false;
```

Comment:
This can also be `return !/[\*\?]/.test(lastString)`, but I think the current one reads better.